docs/development/extensions-core/druid-basic-security.md

```diff
@@ -182,24 +333,131 @@ druid.auth.authorizer.MyBasicMetadataAuthorizer.type=basic
+
+The initial admin user with role defined in initialAdminRole property if specified, otherwise the default admin role will be assigned.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: admin
+
+**`druid.auth.authorizer.MyBasicLDAPAuthorizer.initialAdminRole`**
+
+The initial admin role to create if it doesn't already exists.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: admin
+
+**`druid.auth.authorizer.MyBasicLDAPAuthorizer.initialAdminGroupMapping`**
+
+The initial admin group mapping with role defined in initialAdminRole property if specified, otherwise the default admin role will be assigned. The name of this initial admin group mapping will be set to adminGroupMapping<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: null
+
+**`druid.auth.authorizer.MyBasicLDAPAuthorizer.roleProvider.type`**
+
+The type of role provider (ldap) to authorize requests credentials.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: metadata
+
+**`druid.auth.authorizer.MyBasicLDAPAuthorizer.roleProvider.groupFilters`**
+
+Array of LDAP group filters used to filter out the allowed set of groups returned from LDAP search. Filters can be begin with *, or end with ,* to provide configurational flexibility to limit or filter allowed set of groups available to LDAP Authorizer.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: null
+
+#### Properties for LDAPS
+
+Use the following properties to configure Druid authentication with LDAP over TLS (LDAPS). See [Configure LDAP authentication](../../operations/auth-ldap.md) for more information.
+
+**`druid.auth.basic.ssl.protocol`**
+
+SSL protocol to use. The TLS version is 1.2.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: Yes<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: tls
+
+**`druid.auth.basic.ssl.trustStorePath`**
+
+Path to the trust store file.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: Yes<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: N/A
+
+**`druid.auth.basic.ssl.trustStorePassword`**
+
+Password to access the trust store file.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: Yes<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: N/A
+
+**`druid.auth.basic.ssl.trustStoreType`**
+
+Format of the trust store file. For Java the format is jks.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: jks
+
+**`druid.auth.basic.ssl.trustStoreAlgorithm`**
+
+Algorithm used by the trust manager to validate certificate chains.<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Required**: No<br>
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;**Default**: N/A
+
+**`druid.auth.basic.ssl.trustStorePasswordProvider`**
```

Comment:
In my understanding this property  should be `druid.auth.basic.ssl.trustStorePassword`  variable name is `trustStorePasswordProvider`

Reply:
@tijoparacka I don't understand your comment. Can you please add the suggested change to this section?

Reply:
Updated.